src/main/java/Initiator.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * The initiator of Duke. Runs appropriate commands with user inputs
+ */
+public class Initiator {
+    /**
+     * Takes in a line of user input.
+     * If command is valid, return command accordingly.
+     * Throws DukeException if command is invalid.
+     *
+     * @param commands The commands from the user.
+     * @return appropriate commands.
+     * @throw DukeException if command is invalid.
+     */
+    public static Object InitiatorCommand(String commands) throws DukeException {
+        if (commands == "list") {
+            return new List();
+        } else if (commands == "mark") {
```

Comment:
nitpicking but will it adhere more to the coding standard if the argument name is "command" instead of "commands"?

Reply:
agreed